src/languageTools/DefaultProviders.js

```diff
@@ -335,16 +335,21 @@ define(function (require, exports, module) {
 
     function LintingProvider() {
         this._results = new Map();
+        this._defferedLintingPromise = new Map();
```

Comment:
Nit: the member variables have issues:

- deffered => deferred (typo)
- change defferedLintingPromise to something which seems more intuitive for a map (suggestion: use 'promises' or 'promiseMap' in the name)
- V => v (lowercase, only Modules and Classed are capitalized by convention)
- lintingName => could be changed to 'providerName' or 'linterName'

Reply:
fixed